files/conf/docker-daemon.json

```diff
@@ -0,0 +1,7 @@
+{
+    "registry-mirrors": [
+        {{- range $i, $e := .RegistryMirrors }}
```

Comment:
![download (9)](https://user-images.githubusercontent.com/1071648/86360052-83b60e80-bc7a-11ea-8f80-ef6aa2b0c299.png)
 is it only for me formatting looks like this?

Reply:
It's for everyone. This is probably because it isn't a valid JSON before templating.